packages/server/tsconfig.json

```diff
@@ -4,6 +4,18 @@
     "require": ["tsconfig-paths/register"],
     "swc": true
   },
-  "include": ["src/**/*", "specs", "__mocks__"],
+  "include": [
```

Comment:
This should not be necessary if the packages are accessed via the `@budibase/shared-core` path.